var/spack/repos/builtin/packages/openfast/package.py

```diff
@@ -81,4 +84,7 @@ def cmake_args(self):
                     '-DHDF5_USE_STATIC_LIBRARIES=ON',
                 ])
 
+        if sys.platform == 'darwin':
```

Comment:
(I'm fine to merge without this being changed, but for future reference) Ideally this would test `spec.architecture`. Testing `sys.platform` is required for setting default variant values. Where possible (inside of package methods), it is best to check the spec.

(same goes for Tioga)

Reply:
Ah, ok. I will make these changes before merging.

Reply:
Thanks! And FYI 3 or 4 of the packages have this behavior (just clarifying since in the above comment I mentioned just `openfast` and `Tioga`)

Reply:
Thanks. I committed a change I think resolves this.